src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -811,6 +868,14 @@ void ov::npuw::LLMInferRequest::infer_generate(ov::SoPtr<ov::ITensor> input_ids,
         OPENVINO_THROW("KV-Cache is full.");
     }
 
+    if (auto sliding_mask_port = m_kvcache_in_ports.find(layer_names::gemma_sliding_mask);
+        sliding_mask_port != m_kvcache_in_ports.end()) {
+        // Fill once and update on each iteration instead?
```

Comment:
It seems right, could you please add `FIXME` or `TODO`?

Reply:
Done